src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java

```diff
@@ -497,7 +497,7 @@ public DataFile findCheapAndEasy(Long id) {
         // If content type indicates it's tabular data, spend 2 extra queries 
         // looking up the data table and tabular tags objects:
         
-        if (MIME_TYPE_TSV.equalsIgnoreCase(contentType)) {
+        if (MIME_TYPE_TSV.equalsIgnoreCase(contentType) || MIME_TYPE_TSV_ALT.equalsIgnoreCase(contentType)) {
```

Comment:
If I understand correctly (e.g. from the code links below), MIME_TYPE_TSV_ALT should be the only mimetype that will have variable metadata. So -  can you also drop the first MIME_TYPE_TSV in this if statement?

Ingested tabular file mimetype set to MIME_TYPE_TSV_ALT:
https://github.com/IQSS/dataverse/blob/0bcdcfd09b6e95c52bd9a8598fc4a99511cb7e84/src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java#L922

Incoming files of type MIME_TYPE_TSV_ALT changed to MIME_TYPE_TSV
https://github.com/IQSS/dataverse/blob/0bcdcfd09b6e95c52bd9a8598fc4a99511cb7e84/src/main/java/edu/harvard/iq/dataverse/ingest/IngestServiceBean.java#L355-L358

Reply:
I thought about it, but I believe it is entirely possible for some legacy installations to still have some ingested tabular files with the other mime types. So I figured I would leave it in place to be safe. The worst possible drawback here is 1 wasted custom query, if this not an ingested tab. file. 
(We make sure NOT to rely on the mime type, for identifying tabular data files elsewhere, and instead check for the presence of the DataTable object; but this is one situation where we have to - because we only have the search result, but not the entity).